Since the ifort compiler only recognizes .f90 extensions as free format code,
this .f95 file requires explicit free-form compiler directive
Author: Davide Vanzo (Vanderbilt University)
diff -ru tseries.orig/src/cfuncs.f95 tseries/src/cfuncs.f95
--- tseries.orig/src/cfuncs.f95	2019-07-16 15:39:24.086410749 -0500
+++ tseries/src/cfuncs.f95	2019-07-16 15:39:38.222411140 -0500
@@ -1,3 +1,4 @@
+!DIR$ FREEFORM
 module cfuncs
   interface
      subroutine cnlprt(msg, plen) bind(C, name = 'cnlprt_C')
